### PR TITLE
edited technical specifications, added data models

### PR DESCRIPTION
**Pull request by MichalMed**
https://github.com/kbss-cvut/termit-web/issues/4

### CODE REVIEW

File: _pages/technical-details.markdown [hunk 1]
```diff
@@ -62,9 +64,45 @@ of a Semantic Web-based information system as smooth as possible.
...
+
+### Data description ontology
+
+Basic ontology used for description of any data is called Data description ontology and is described on [this page](http://onto.fel.cvut.cz/ontologies/slovnik/agendovy/popis-dat/current/index-en.html). It is based on the Unified Foundational Ontology (UFO).
+
+The current version of Data description ontology can be found [here](https://onto.fel.cvut.cz/ontologies/page/slovnik/agendovy/popis-dat/model/verze/1.0.1) in turtle or RDF/XML.
+
+Following sections describes basic classes defined in the model.
+
+#### Vocabulary
```
Comment (psiotwo) [inline _pages/technical-details.markdown:84]: Maybe a diagram would be more informative than this extract of classes. Example would also help IMO.

File: _pages/technical-details.markdown [hunk 2]
```diff
@@ -62,9 +64,45 @@ of a Semantic Web-based information system as smooth as possible.
...
+Glossary is a list of all terms present in vocabulary, using only skos concepts.
+
+#### Model
+Model brings ontological relations to the terms defined in the glossary, using mainly UFO.
+
+#### Term
+Term is any concept used in the vocabulary.
+
+#### Attribute
+Attribute is a special type of term, representing a dependent property. It may represent data attributes, reference attributes or statistical attributes.
```
Comment (psiotwo) [inline _pages/technical-details.markdown:97]: Will the reader understand what 'dependent property' is?

File: _pages/technical-details.markdown [hunk 3]
```diff
@@ -62,9 +64,45 @@ of a Semantic Web-based information system as smooth as possible.
...
+
+#### Source
+Source represents the origin of data that are being modelled, e.g. database, data set, document, file etc...
+
+### TermIt ontology
+
+TermIt ontology is created in the process of creation of TermIt software. It inherits from common Data description ontology and extends it for the concepts used specifically in TermIt.
```
Comment (psiotwo) [inline _pages/technical-details.markdown:104]: TermIt ontology has been created while developing TermIt.

**Review by psiotwo — COMMENTED** [review]
see comments

File: _pages/technical-details.markdown (hunk 1 above)
Comment (MichalMed) [reply _pages/technical-details.markdown:84]: Full documentation of a model is included via link. I was thinking about diagram, but full diagram is too complicated and simple does not connect chosen classes directly.

File: _pages/technical-details.markdown (hunk 2 above)
Comment (MichalMed) [reply _pages/technical-details.markdown:97]: I am not sure. Do you have better idea how to describe that? I spent over ten minutes with this :)

Comment (psiotwo) [reply _pages/technical-details.markdown:97]: Attribute types are terms (e.g. `ship length`, `person name`). An attribute belongs to an object and cannot exist without it.

**Pull request merged**